clients/venice-push-job/src/main/java/com/linkedin/venice/vpj/VenicePushJobConstants.java

```diff
@@ -112,18 +112,20 @@ private VenicePushJobConstants() {
   public static final String KAFKA_INPUT_BROKER_URL = "kafka.input.broker.url";
   // Optional
   public static final String KAFKA_INPUT_MAX_RECORDS_PER_MAPPER = "kafka.input.max.records.per.mapper";
+
+  // Legacy prefix kept for backward compatibility
+  public static final String KIF_RECORD_READER_KAFKA_CONFIG_PREFIX = "kif.record.reader.kafka.";
+
   /**
    * The default max records per mapper, and if there are more records in one topic partition, it will be
    * consumed by multiple mappers in parallel.
    * BTW, this calculation is not accurate since it is purely based on offset, and the topic
    * being consumed could have log compaction enabled.
    */
-  public static final long DEFAULT_KAFKA_INPUT_MAX_RECORDS_PER_MAPPER = 5000000L;
-
-  public static final String KIF_RECORD_READER_KAFKA_CONFIG_PREFIX = "kif.record.reader.kafka.";
+  public static final long DEFAULT_PUBSUB_INPUT_MAX_RECORDS_PER_MAPPER = 5000000L;
```

Comment:
* nit
you can write it as `5_000_000` more readable and standard practice in Java

Reply:
Done